plugin/types.go

```diff
@@ -5,8 +5,13 @@ type plugin struct {
 }
 
 type DroneConfig struct {
-	Kind  string `yaml:"kind" json:"kind"`
-	Type  string `yaml:"type" json:"type"`
-	Name  string `yaml:"name" json:"name"`
-	Data    []byte `yaml:"-" json:"-"`
+	Kind string `yaml:"kind" json:"kind"`
+	Type string `yaml:"type" json:"type"`
+	Name string `yaml:"name" json:"name"`
+	Data []byte `yaml:"-" json:"-"`
+}
+
+type Promotion struct {
+	Build  int64  `json:"build,omitempty"`
+	Deploy string `json:"deploy,omitempty"`
```

Comment:
**nit:** It probably doesn't matter for our purposes, but this marshals from `deploy_to` in the drone request.

Reply:
Renamed it for consistency, the build struct calls it `Deploy`.